tests/test_telemetry.py

```diff
@@ -42,6 +43,29 @@ def validate_agent_telemetry_data(self, validator, success_by_default=False):
         for data in telemetry_data:
             validator(data)
 
+    def test_telemetry_message_data_size(self):
+        """Test telemetry message data size"""
+
+        def validator(data):
+            if sys.getsizeof(data) / 1000000 >= 5:
```

Comment:
It gives the size in memory in python, is it really what the RFC says ? FYI, `data["request"]["length']` gives the size in bytes of the actual HTTP request.